### ISSUE TITLE
Make race result more visible

### ISSUE THREAD

**Issue by Wuzzy2** [juzzlin/DustRacing2D#73]
This is a suggestion.

Currently, it's not quite clear to see on which rank you finished. This can be a bit confusing when the race was very tight.

I suggest this:

When the player finished the race, display a LARGE message in the center that shows the result. The rationale being that the race end is an important event, so it should be highly visible.

The exact behaviour depends on the game mode.

## One player race:

Show the player's rank: “1st place”, “2nd place”, “3rd place”, etc.

## Two player race / duel:
Same as in one player race, but display the rank of each player on each splitscreen separately.

## Time trial:
Show “The Time Trial has ended!” in center instead of the bottom. (basically, move the existing message)

## Additional suggestions:

* Don't display “The winner has finished the race” in singleplayer race, when the player is the winner. This message is a little irritating if you won.

**Comment by juzzlin** [comment]
Yeah, I've been thinking about something like this, too. IIRC I didn't do this, because there's no way to set player names and it wouldn't be that nice to just display "Player 1", "Player 2", "Computer 1"..., but I guess that would still be better than nothing.

**Comment by Wuzzy2** [comment]
Well, as I said, just display “1st place”, “2nd place”, “3rd place” etc. That's more straightforward and doesn't require you to introduce names at all.

**Comment by juzzlin** [comment]
Oh, right. I was thinking about a full standings board.
